docs/api/helpers/pipe.md

```diff
@@ -63,7 +63,7 @@ Close the communication channel
  | in_path() -> str
 ```
 
-Returns the rendezvous point for incoming communication
+Rendezvous point for incoming communication
```

Comment:
This needs to be auto generated: `python scripts/generate_api_docs.py` or just leave it unchanged (we always run the script before releases anyway)